google-cloud-network_connectivity-v1/.repo-metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+    "api_id": "networkconnectivity.googleapis.com",
+    "client_documentation": "https://googleapis.dev/ruby/google-cloud-network_connectivity-v1/latest",
+    "distribution_name": "google-cloud-network_connectivity-v1",
+    "language": "ruby",
+    "name": "networkconnectivity",
+    "name_pretty": "Network Connectivity V1 API",
+    "product_documentation": "https://cloud.google.com/network-connectivity/docs",
```

Comment:
This URL redirects to `https://cloud.google.com/network-connectivity/docs/concepts`, but I guess that's fine.